fe/fe-core/src/test/java/com/starrocks/pseudocluster/Tablet.java

```diff
@@ -271,7 +277,8 @@ public synchronized void cloneFrom(Tablet src) throws Exception {
             commitRowset(p.second.copy(), p.first);
         }
         cloneExecuted.incrementAndGet();
-        LOG.info("tablet:{} clone src:{} before:{} after:{}", id, src.versionInfo(), oldInfo, versionInfo());
+        System.out.printf("\ntablet:%s clone src:%s before:%s after:%s\n", id, src.versionInfo(), oldInfo, versionInfo());
```

Comment:
I know CI environment only show stdout/errr and delete log files after test run, so you want to print to stdout. There is lots of other information in the log, so putting it in log is better for cross-checking things. 
Currently, how about both log and stdout?

Reply:
ok